### PR TITLE
Add LibreLinkUp Plugin

### PR DESCRIPTION
**Pull request by FloSchl8**
using LibreLinkUp API to display GCM values
grafik

### CODE REVIEW

File: Lifestyle/librelinkup.1m.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+# your region/country
+# available: us, eu, de, fr, jp, ap, au, ae
+country = "eu"
+
+# used for ordering multiple patients
+# the value from this id will be displayed in the menu bar first, all others can be seen in the dropdown
+# all IDs will also be shown after the glucose values
+# if you have only one patient you can leave this blank
+first_patient = ""
```
Comment (sprak3000) [inline Lifestyle/librelinkup.1m.py:27]: These fit the use case of [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html).

**Review by sprak3000 — CHANGES_REQUESTED** [review]
@FloSchl8 

Could you

1. Run `chmod +x` on the script file.
2. Add a `<xbar.image>` metadata tag linking to a screenshot of the plugin in action.

Also, consider using [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) for the email, password, etc. configuration items.

File: Lifestyle/librelinkup.1m.py (hunk 1 above)
Comment (FloSchl8) [reply Lifestyle/librelinkup.1m.py]: done :)

**Comment by FloSchl8** [comment]
> @FloSchl8
> 
> Could you
> 
>     1. Run `chmod +x` on the script file.
> 
>     2. Add a `<xbar.image>` metadata tag linking to a screenshot of the plugin in action.
> 
> 
> Also, consider using [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) for the email, password, etc. configuration items.

All done :)

**Pull request merged**